src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,81 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final byte OPERATION_TYPE_INDEX = 0;
+    private static final byte AMOUNT_INDEX = 1;
+    private String fromFileName;
+    private String toFileName;
+
+    public String getFromFileName() {
+        return fromFileName;
+    }
+
+    public String getToFileName() {
+        return toFileName;
+    }
```

Comment:
You don't need to create this fields, just put it as method parameters in readFile(String fromFileName), createReport(String fromFileName)